client/src/components/Lists/PitchTeams/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch, IUser } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useTeams } from '../../../contexts';
+
+interface PitchTeamsProps {
+  pitch: IPitch;
+  user?: Pick<IUser, '_id'>;
+  assignmentContributors?: boolean;
+}
+
+const PitchTeams: FC<PitchTeamsProps> = ({
+  pitch,
+  user,
+  assignmentContributors = false,
+}): ReactElement => {
+  const { getTeamFromId } = useTeams();
+
+  const getPitchTeams = (): string[] => {
+    if (user) {
+      let contributor: { teams: string[]; [x: string]: any } | undefined;
```

Comment:
This type is kind of hard to read. I see what it does though. I went onto your branch and typescript doesn't seem to complain if you take out the `[x: string]: any`.